samples/open-telemetry/loki/REAME.md

```diff
@@ -0,0 +1,91 @@
+# Open Telemetry ALS with Loki
```

Comment:
Consider describing what ALS is, e.g., Access Log Service (ALS)(https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/access_loggers/grpc/v3/als.proto).